exonum-light-client/src/main/lombok/com/exonum/client/response/InstanceSpec.java

```diff
@@ -20,7 +20,7 @@
 import lombok.Value;
 
 @Value
-public class ServiceInfo {
+public class InstanceSpec {
```

Comment:
I am not sure `InstanceSpec` works well because 'instance' is a rather generic term (any object is an _instance_ of some class). `ServiceInstanceSpec`? Or `ServiceInstanceInfo` for the spec for a service instance in other contexts also includes its initial configuration?

Reply:
I used the same naming as core does so that it's easier to look up the corresponding class if needed (for example, when checking if there were any API updates). I could move those to `client.response.service` package so that naming is more evident from the package name?